firebase-vscode/src/data-connect/language-client.ts

```diff
@@ -75,38 +78,70 @@ export function setupLanguageClient(context) {
     "graphQLlanguageServer",
     "GraphQL Language Server",
     serverOptions,
-    clientOptions
+    clientOptions,
   );
 
   // register commands
   const commandShowOutputChannel = vscode.commands.registerCommand(
     "fdc-graphql.showOutputChannel",
-    () => outputChannel.show()
+    () => outputChannel.show(),
   );
 
   context.subscriptions.push(commandShowOutputChannel);
 
+  const generateYamlFile = async () => {
+    const basePath = vscode.workspace.rootPath;
+    const filePath = ".firebase/.graphqlrc";
+    const fileUri = vscode.Uri.file(`${basePath}/${filePath}`);
+    const folderPath = ".firebase";
+    const folderUri = vscode.Uri.file(`${basePath}/${folderPath}`);
+
+    // TODO: Expand to multiple services
+    const config = configs.values[0];
```

Comment:
Would you mind short-circuiting this when `configs.values.length === 0`? Otherwise it throws. Feel free to put in an empty config